tools/apiview/parsers/rust-api-parser/ci.yml

```diff
@@ -0,0 +1,114 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/apiview/parsers/rust-api-parser
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/apiview/parsers/rust-api-parser
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: 'Build'
+        variables:
+          - template: /eng/pipelines/templates/variables/image.yml
+          - name: NodeVersion
+            value: '18.x'
+          - name: ArtifactName
+            value: 'apiview'
+          - name: RustGeneratorDirectory
+            value: 'tools/apiview/parsers/rust-api-parser'
+        jobs:
+          - job: 'Build'
+
+            pool:
+              name: $(LINUXNEXTPOOL)
+              image: $(LINUXNEXTVMIMAGE)
+              os: linux
+
+            steps:
+              - task: NodeTool@0
+                inputs:
+                  versionSpec: '$(NodeVersion)'
+                displayName: 'Use NodeJS $(NodeVersion)'
+
+              - script: |
+                  npm install -g npm@8.16.0
+                displayName: "Install npm 8.16.0"
```

Comment:
Why does the version matter? This creates a maintenance issue if we upgrade node. Already, we've deprecated v18 for Azure SDKs so you're already behind.

Reply:
copied this from the JS parser, I agree, we should change this.